### DISCUSSION TITLE
Ward BRDF

### DISCUSSION THREAD

**Discussion by alakhag** [mitsuba-renderer/mitsuba3#740] [Q&A]
## Summary

I see that the previous version included Ward BRDF, but the new version only contains Disney BRDF. Can you guide how to utilize Ward BRDF model?

## System configuration

System information:

  OS: Ubuntu 20.04.4 LTS
  CPU: Intel(R) Core(TM) i7-9700K CPU @ 3.60GHz
  GPU: NVIDIA GeForce RTX 2080 Ti
NVIDIA GeForce RTX 2080 Ti
  Python: 3.8.3 (default, Jul  2 2020, 16:21:59) [GCC 7.3.0]
  NVidia driver: 470.141.03
  CUDA: 11.0.221
  LLVM: 12.0.0

  Dr.Jit: 0.4.1
  Mitsuba: 3.2.1
     Is custom build? False
     Compiled with: GNU 10.2.1
     Variants:
        scalar_rgb
        scalar_spectral
        cuda_ad_rgb
        llvm_ad_rgb

**Comment by njroussel** [comment]
Hi @alakhag 

Indeed, the Ward BSDF was never ported to Mitsuba3. What do you mean by "guide how to utilize"? Mitsuba 3 doesn't have some sort of mechanism that allows old plugins to be imported. The plugin would need to be re-implemented in Mitsuba3.

**Reply by wjakob** [reply]
@alakhag  You are welcome to port the Ward plugin and propose a PR. But this is not high on our priority list (we consider the Ward model to be a legacy BRDF model that should likely be replaced with more recent physically-based variants based on microfacet theory.)
